build.sh

```diff
@@ -119,7 +119,7 @@ case $TASK in
                 base_eg=$( basename ${example} )
                 printf "\t %s\n" ${base_eg}
             done
-            exit 1
+            exit 0
```

Comment:
why don't we want to exit with a failure here? seems like `exit 1` is correct tbh. same below